compiler/Makefile

```diff
@@ -164,9 +164,15 @@ $(CHPL): $(CHPL_OBJS) $(CHPL_CONFIG_CHECK) | $(CHPL_BIN_DIR)
 	$(TAGS_COMMAND)
 	$(EBROWSE_COMMAND)
 
-$(CHPLDOC): $(CHPL)
+$(CHPLDOC): $(CHPL) dyno-chpldoc
+	rm -f $(CHPLDOC)-legacy
+	ln -s $(notdir $(CHPL)) $(CHPLDOC)-legacy
```

Comment:
This is OK but IMO it would be nicer to have these rm/ln commands in a separate CHPLDOC-legacy rule

Reply:
I added a separate target for $(CHPLDOC)-legacy